src/OmniSharp.Abstractions/Models/v2/BlockSpan.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace OmniSharp.Models.V2
+{
+    public class BlockSpan
```

Comment:
Should this be more specific to the purpose its indeed? E.g. `CodeFoldingBlock` or something like that?